config/src/main/java/org/axonframework/config/EventProcessingConfiguration.java

```diff
@@ -83,30 +78,6 @@ default <T extends EventProcessor> Optional<T> eventProcessor(String name, Class
         return eventProcessor(name).filter(expectedType::isInstance).map(expectedType::cast);
     }
 
-    /**
-     * Obtains an {@link EventProcessor} by it's {@code processingGroup}.
-     *
-     * @param processingGroup a {@link String} specifying the processing group of an {@link EventProcessor}
-     * @param <T>             the type of the expected {@link EventProcessor}
-     * @return an {@link Optional} referencing the {@link EventProcessor}
-     */
-    <T extends EventProcessor> Optional<T> eventProcessorByProcessingGroup(String processingGroup);
```

Comment:
If you could add 3 to 5 lines that we have remove the Processing Group layer entirely to the `api-changes.md`, then we cover the removal of these methods too.

Reply:
Done :)